.github/workflows/servicenow.yaml

```diff
@@ -31,9 +33,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+    - uses: actions/checkout@v4
     - name: Set up Python 3.x, latest minor release
-      uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38
+      uses: actions/setup-python@v5
```

Comment:
There was a supply chain attack a few months ago based on which we switched to commit SHAs based on https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Ah, I didn't know that, I will add back the SHAs.